src/components/DriverCard.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+function DriverCard(props) {
+    const {name, rating, img, car} = props
+  return (
+    <div>
+      <div
+        style={{
+          paddingLeft: '150px',
+          paddingRight: '150px',
+          width: '300px',
+          height: '100px',
+          backgroundColor: 'blue',
+          borderRadius: '12px',
+          color: 'white',
+          display: 'flex',
+          fontSize: '17px',
+          margin: '30px',
+          justifyContent: 'space-around',
+        }}
+      >
+        <img
+          src={img}
+          alt=""
+          style={{ width: '80px', height: '80px', borderRadius: '60px', paddingTop: '10px' }}
+        />
+        <div style={{ display: 'flex'}}>
+          <p>{name}</p>
+          <p>{rating}</p>
```

Comment:
Here you should re-use the `Rating` component that you've already done.

So instead of this:
```
<p>{rating}</p>
```

You should do this:
```
<Rating>{rating}</Rating>
```